### DISCUSSION TITLE
👍 Thumbs-up matter!

### DISCUSSION THREAD

**Discussion by mestiez** [studio-minus/ppg-bugs-and-requests#2011] [Announcements]
I have said in the past that I don't really care about the thumbs-up 👍 reaction on requests or other issues but that has changed. There are now so many issues that I'm having trouble looking at them all and remembering the ones I find interesting.

If you really like a feature request give it a thumbs-up, it increases the chances that I will implement it.

**Comment by TeslaCoil9001** [comment]
👍👍👍👍👍

**Comment by Reycko** [comment]
👍👍👍👍👍👍👍👍👍👍👍👍👍👍👍
